### ISSUE TITLE
Return a bool indicating success/failure from bv.navigate(...)

### ISSUE THREAD

**Issue by gaasedelen** [Vector35/binaryninja-api#1111]
This is a minor suggestion, but it would be nice if `bv.navigate` would return a True/False indicating success or failure (of the navigation request).

```
bv.navigate(bv.view, address) --> True/False
```

IDA provides this, eg:

```
Python>idaapi.jumpto(0x1400038D0)
True
```

**Comment by bpotchik** [comment]
Updated. Will close when included in the next build.

**Comment by bpotchik** [comment]
Fixed in build 1.1.1278-dev.
